theme.json

```
@@ -97,6 +97,63 @@
 			],
 			"units": ["%", "px", "em", "rem", "vh", "vw"]
 		},
+		"typography": {
+			"defaultFontSizes": false,
+			"fluid": true,
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Extra Small",
+					"size": "14px",
+					"slug": "x-small"
+				},
+				{
+					"fluid": {
+						"max": "18px",
+						"min": "18px"
```

We may also want to use `rem` for consistency for the fluid values.

```suggestion
						"max": "1.125rem",
						"min": "1rem"
```